pontoon/checks/libraries/pontoon.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+import re
+
+from collections import defaultdict
+
+MAX_LENGTH_RE = re.compile(r'MAX_LENGTH: (\d+)$', re.MULTILINE)
+
+
+def run_checks(entity, string):
+    """
+    Group all checks related to the base UI
+
+    :arg pontoon.base.models.Entity entity: Source entity
+    :arg basestring string: a translation
+    """
+    checks = defaultdict(list)
+    resource_ext = entity.resource.format
+
+    max_length = re.findall(MAX_LENGTH_RE, entity.comment or '')
+    if max_length and len(string) > int(max_length[0]):
+        checks['pErrors'].append(
+            'Translation too long.'
+        )
+
+    # Prevent empty translation submissions if not supported
+    if resource_ext in {'properties', 'ini', 'dtd', 'ftl'} and string == '':
```

Comment:
It's the other way around - these formats DO SUPPORT empty translations.

Reply:
Also, you should remove `ftl` from that list - it doesn't support empty translations in a sense used here (`''`).

Reply:
I'm sorry -> I misinterpeted the javascript :/